src/classes/BDI_SettingsUI_CTRL.cls

```diff
@@ -94,7 +94,7 @@ public with sharing class BDI_SettingsUI_CTRL {
 
                 // this UI is specific to help text settings in current state
                 BDI_Donations donationImportHandler = 
-                    new BDI_Donations(new BDI_DataImportService(true, BDI_FieldMappingHelpText.getInstance()));
+                    new BDI_Donations(new BDI_DataImportService(true, BDI_DataImportService.getDefaultFieldMapping()));
```

Comment:
This is a small comment, but can add a line break so that the display in github is a bit better? I think we have suggestions for how to break lines in the style guide somewhere